package.json

```diff
@@ -10,34 +10,32 @@
         "build:ts": "wireit",
         "build:ts:watch": "wireit",
         "build:types": "wireit",
         "build:watch": "wireit",
-        "changeset-snapshot-publish": "yarn prepublishOnly && yarn changeset version --snapshot && yarn lint:versions --fix && yarn update-version && yarn changeset publish --no-git-tag --tag snapshot",
-        "changeset-publish": "yarn prepublishOnly && yarn changeset version && yarn install && yarn lint:versions --fix && yarn update-version && yarn changeset publish --no-git-tag && yarn push-to-remote && yarn create-git-tag && yarn postpublish",
-        "update-version": "node ./tasks/update-version.js",
+        "changeset-publish": "yarn prepublishOnly && yarn changeset version && yarn install && yarn lint:versions --fix && yarn update-version && yarn changeset publish && yarn postpublish",
+        "changeset-snapshot-publish": "yarn prepublishOnly && yarn changeset version --snapshot && yarn install && yarn lint:versions --fix && yarn update-version && yarn changeset publish --no-git-tag --tag snapshot",
         "chromatic": "chromatic --build-script-name storybook:build # note that --project-token must be set in your env variables",
-        "create-git-tag": "node --no-warnings tasks/create-git-tag.js",
-        "custom-element-json": "node tasks/custom-element-json.js",
-        "docs:analyze": "cem analyze --globs \"packages/**/*.ts\" --exclude \"**/*.d.ts\" --exclude \"**/stories/**\" --exclude \"**/icons/**\" --exclude \"**/elements/**\" --outdir projects/documentation --litelement",
+        "custom-element-json": "yarn custom-element-json:clean && node tasks/custom-element-json.js && yarn custom-element-json:validate",
+        "custom-element-json:clean": "rimraf packages/*/custom-elements.json tools/*/custom-elements.json",
+        "custom-element-json:validate": "node ./tasks/run-check-cem.js",
         "docs:build": "yarn workspace documentation build",
-        "docs:ci": "yarn docs:analyze && run-p docs:production storybook:build && cp projects/documentation/custom-elements.json projects/documentation/dist/storybook",
-        "docs:preview": "yarn docs:analyze && run-p docs:build storybook:build && cp projects/documentation/custom-elements.json projects/documentation/dist/storybook",
+        "docs:ci": "yarn analyze:docs && run-p docs:production storybook:build && cp projects/documentation/custom-elements.json projects/documentation/dist/storybook",
+        "docs:preview": "yarn analyze:docs && run-p docs:build storybook:build && cp projects/documentation/custom-elements.json projects/documentation/dist/storybook",
         "docs:production": "yarn workspace documentation build:production",
         "docs:review": "alex packages/**/*.md",
         "docs:start": "yarn workspace documentation serve --watch",
-        "find": "test -f custom-elements.json",
-        "format:css": "yarn lint:css --fix && pretty-quick --pattern \"{packages,tools}/**/*.css\"",
```

Comment:
I removed this formatter since we're looking to migrate to lint-staged but we can bring back a forced formatting command if we need it.

Reply:
does lint-staged need to be merged in first?

Reply:
No I don't think it does, the `format:css` command isn't called by anything so it's really just a question of if anyone is manually using this one.

Reply:
i have never used it fwiw